types/shared.d.ts

```diff
@@ -121,7 +121,10 @@ export interface StripeConnectWrapper {
 }
 
 export interface StripeConnectInstance {
-  create: (tagName: ConnectElementTagName) => HTMLElement | null;
+  create: (
+    tagName: ConnectElementTagName,
+    reactSdk?: boolean
```

Comment:
Instead of adding this here (which may confuse some platforms), can we add a setter on the connect instance? Something like `setReactSdkAnalytics`, similar to how we call getAppearanceOptions - we can simply call that before create: https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/frontend/connect/submerchant-surfaces/src/connect/Connect.ts#L222. Why I prefer that: adding this parameter on `create` may confuse platforms - why do they need to know about this on their typescript types?

Also, we may add more parameters here, and this `reactSdk` parameter may confuse them.

Separately, did you see the task on package version? Wondering if instead of setting a boolean, we should go straight into the package version approach (if the package version is null, that is enough to know that it isn't being used)

Reply:
That makes sense, I'll make the corresponding changes in sus